.travis.yml

```diff
@@ -21,6 +21,7 @@ jobs:
       env:
         - "JOB_NAME='Ubuntu 19.10, full VM'"
         - "VM_IMAGE=https://cloud-images.ubuntu.com/eoan/current/eoan-server-cloudimg-amd64.img"
+    - python: 3.6.1  # earliest 3.6 version available on Travis
```

Comment:
Is there a specific motivation for this?

Reply:
Not really, I think I just mirrored the fact that we were supporting early versions of Python 3.5. In fact it looks like `test_ki_wakes_us_up` could be simplified if we decide to only support 3.6.2+.